internal/exec/stages/files/files.go

```diff
@@ -60,19 +60,29 @@ func (creator) Name() string {
 type stage struct {
 	util.Util
 	toRelabel []string
+	isLive    bool
 }
 
 func (stage) Name() string {
 	return name
 }
 
+func (s stage) ApplyLive(config types.Config) error {
+	s.isLive = true
```

Comment:
Since nothing outside of `Run` currently references this, would it be cleaner to move it out of stage state, and have both `Run` and `ApplyLive` wrap a helper function that takes `isLive` as an argument?